dub.sdl

```diff
@@ -22,13 +22,13 @@ configuration "library" {
 }
 
 configuration "library-nonet" {
-	dependency "vibe-d:http" version=">=0.7.30 <=0.9.0" optional=true
+	dependency "vibe-d:http" version=">=0.7.31 <=0.9.0" optional=true
```

Comment:
Hitting 17131 would pose a natural limit, but it could still be that someone, for some reason, is stuck on vibe.d 0.7.30 while using an older compiler. While it would be even nicer to actually test the compatibility, it seems just like an unnecessary restriction to *require* 0.7.31.

The recipe is not strictly kept in sync with the selections, but I did bump the numbers in the recipe from time to time, basically when the numbers became too old to be sure that it would still work without testing it.